_posts/2022-04-07-launching-open-source-operator-for-openSearch.markdown

```diff
@@ -0,0 +1,46 @@
+Launching Open-Source Operator for OpenSearch 
+----------------------------------------------
+
+### With powerful capabilities for orchestration & management of OpenSearch clusters. The Operator is led by Opster with partners including AWS, SUSE Rancher, Maibornwolff, Logz.io and more.
+
+Overview
+--------
+
+The [OpenSearch Operator](https://github.com/Opster/opensearch-k8s-operator) is fully open-source, licensed as Apache 2.0, and is used for automating the deployment, provisioning, management, and orchestration of OpenSearch clusters and OpenSearch dashboards.
+
+The Operator development is being led by [Opster](https://opster.com/) with partners including [SUSE Rancher](https://www.suse.com/), [Maibornwolff](https://www.maibornwolff.de/en), [AWS](https://aws.amazon.com/), [Logz.io](https://logz.io/) and more.
+
+Operator Capabilities
+---------------------
+
+The Operator enables high-level API use, for easily running advanced OpenSearch operations on Kubernetes. With the Operator, scaling up and down, version upgrades, rolling restarts, securing deployments and managing certificates is simplified and streamlined.
+
+The Operator allows for management of multiple OpenSearch clusters and OpenSearch Dashboards. The Operator [simplifies] operation by providing node draining, adjustment of memory and disk resources on the nodes, advanced shard allocation strategies and auto-scaling based on usage load and resources.
+
+Getting Started
+---------------
+
+In order to install the Operator, follow the steps below.
+### Installing the Operator
```

Comment:
Instead of building, could we point here to the release artifacts?  https://github.com/Opster/opensearch-k8s-operator/releases/tag/v0.9
Builds break all the time, release artifact might be better as it is tested.

Reply:
Added a gz file with helm for release installation.
Also added instruction how to use it.